cnxepub/tests/test_models.py

```diff
@@ -103,67 +103,6 @@ def test_document_attribs(self):
         self.assertEqual(document.metadata['version'], '2')
 
 
-class PrivateUtilitiesTestCase(unittest.TestCase):
-
-    def test_sanitize_xml(self):
```

Comment:
Hm... the function being tested here originally was added to fix another bug that was **also** causing missing content. To simply delete that function and these tests without replacing them with other types of tests (like integration tests that somehow test for no missing content due to html fragments or whatever was causing this older bug) seems **too risky** to me; we could end up with missing content again.